### DISCUSSION TITLE
Cannot send email to gmail due to PTR

### DISCUSSION THREAD

**Discussion by huynhphuong10284** [postalserver/postal#2131] [Help with using Postal]
I've tried to send email to gmail (just for test some emails, not spam) and always see the following error message:

550-5.7.25 [2a09:2dc0:0:58::1] The IP address sending this message does not have 550-5.7.25 a PTR record setup, or the corresponding forward DNS entry does not 550-5.7.25 point to the sending IP. As a policy, Gmail does not accept messages 550-5.7.25 from IPs with missing PTR records. Please visit 550-5.7.25 https://support.google.com/mail/answer/81126#ip-practices for more 550 5.7.25 information. c19-20020a2ebf13000000b0025e0f597a6csi4584150ljr.567 - gsmtp

I've worked with cloudns as well as VPS provider to configure revers dns, PTR record and they confirmed all configurations are valid. I don't know the next steps for work. Could you please input your ideas on this case.

Thanks,
Phuong

**Comment by willpower232** [comment]
You could use something like mail-tester.com or mxtoolbox.com to double check that your PTR records are valid and working.

**Comment by huynhphuong10284** [comment]
Yes, I've worked with VPS provider and cloudns for checking setting up. Everything about PTR records are valid. Beside that, I also test with some commands as the following and it works.

 dig +trace -x 2a09:2dc0:0:58::3

  dig +short AAAA mail.kloviya.com

My IP range from 2a09:2dc0:0:58::1 - 2a09:2dc0:0:58::2300, could you please have a look?

Thanks

**Reply by willpower232** [reply]
the IPv4 looks good to me, did you try mail-tester.com to see what it thinks about your messages?

**Reply by huynhphuong10284** [reply]
mail-tester.com only supports IPv4. If we us IPv6, it says "Address family not supported by protocol - bind(2) for "2a09:2dc0:0:58::988" port". However, if I switch to IPv4 - my system works well but we need IPv6 due to my system has 10K IPv6 free

**Reply by willpower232** [reply]
Ah I see, you're not answering SMTP on both IPv4 and IPv6 which is probably upsetting Google and also potentially going to make it difficult for you to receive mail and bouncebacks

**Reply by huynhphuong10284** [reply]
Yes, I'm trying to use IPv6 only for my SMTP and I think it's a problems from Postal. Do you have any ideas to fix?

**Reply by willpower232** [reply]
Either you have to start answering SMTP on IPv4 or you have to remove the PTR A record so your server is only referenced by AAAA / IPv6.

I'm not sure how well cutting yourself off from the IPv4 will go so I'd suggest using iptables or something to route port 25 on your IPv4 address to Postal.

**Comment by kayastha1988** [comment]
Hi, this is also something similar I am facing after successful installation of Postal SMTP. I tried sending email to different email ids but in gmail only, I am getting issue. Email was not delivered to gmail. Later, I find out that my domain/IP does not have any PTR records.

How can I add this PTR records to my dns? Doesnt sounds like related question but hoping for positive answer.

Thanks
